sycl/unittests/queue/GetProfilingInfo.cpp

```diff
@@ -429,18 +416,54 @@ TEST(GetProfilingInfo, partial_profiling_workaround) {
     cgh.parallel_for<InfoTestKernel>(globalWIs, [=](sycl::id<1> idx) {});
   });
   event.wait();
-  try {
-    event.get_profiling_info<sycl::info::event_profiling::command_submit>();
-    FAIL() << "No exception was thrown";
-  } catch (sycl::exception &e) {
-    EXPECT_EQ(e.code(), sycl::errc::invalid);
-    EXPECT_STREQ(
-        e.what(),
-        "Submit profiling information is temporarily unsupported on this "
-        "device. This is indicated by the lack of queue_profiling aspect, but, "
-        "as a temporary workaround, profiling can still be enabled to use "
-        "command_start and command_end profiling info.");
-  }
-  event.get_profiling_info<sycl::info::event_profiling::command_start>();
-  event.get_profiling_info<sycl::info::event_profiling::command_end>();
+  auto submit_time =
+      event.get_profiling_info<sycl::info::event_profiling::command_submit>();
+  auto start_time =
+      event.get_profiling_info<sycl::info::event_profiling::command_start>();
+  auto end_time =
+      event.get_profiling_info<sycl::info::event_profiling::command_end>();
+  (void)submit_time;
+  (void)start_time;
+  (void)end_time;
+}
+
+TEST(GetProfilingInfo, fallback_profiling_PiGetDeviceAndHostTimer_unsupport) {
+  sycl::unittest::PiMock Mock;
+  sycl::platform Plt = Mock.getPlatform();
+  Mock.redefineBefore<sycl::detail::PiApiKind::piEventGetProfilingInfo>(
+      redefinedPiEventGetProfilingInfo);
+  Mock.redefine<sycl::detail::PiApiKind::piGetDeviceAndHostTimer>(
+      redefinedFailedPiGetDeviceAndHostTimer);
+  Mock.redefineAfter<sycl::detail::PiApiKind::piDeviceGetInfo>(
+      redefinedDeviceGetInfoAcc);
+  const sycl::device Dev = Plt.get_devices()[0];
+  sycl::context Ctx{Dev};
+
+  // support with FallbackProfiling
+  // ASSERT_FALSE(Dev.has(sycl::aspect::queue_profiling));
+
+  static sycl::unittest::PiImage DevImage_1 =
```

Comment:
```suggestion
  static sycl::unittest::PiImage DevImage =
```
There is no `DevImage_2` in the code